plugins/ua_pubsub_udp.c

```diff
@@ -18,6 +18,7 @@
 typedef struct {
     int ai_family;                    /* Protocol family for socket. IPv4/IPv6 */
     struct sockaddr_storage *ai_addr; /* https://msdn.microsoft.com/de-de/library/windows/desktop/ms740496(v=vs.85).aspx */
```

Comment:
I would prefer if both `sockaddr_storage` were not pointers but direct members.
That removes a lot of the allocation and cleanup logic.

Reply:
I agree. I will update the PR and remove the pointers from this struct.